ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/PubsubMessageToTableRow.java

```diff
@@ -202,7 +201,7 @@ private TableRow payloadTableRow(PubsubMessage message, TableReference ref, Stri
           }
         });
       } catch (ExecutionException e) {
-        throw new BubbleUpException(e.getCause());
+        throw new RuntimeException(e);
```

Comment:
I think this should be `UncaughtExecutionException(e.getCause())` and the same below.